kahuna/public/js/upload/jobs/required-metadata-editor.js

```diff
@@ -153,36 +153,10 @@ jobs.controller('DescriptionPlaceholderCtrl',
                 ['$scope',
                  function($scope) {
 
-    var people = [
-        'George Osborne',
-        'A teary Nick Clegg',
-        'Pop singer Rihanna',
-        'US actress and director Angelina Jolie',
-        'George W. Bush'
-    ];
-
-    var actions = [
-        'eating',
-        'caught with',
-        'wrestling',
-        'chants while marching for a third night of protests about',
-        'making a toast to'
-    ];
-
-    var things = [
-        'a large pheasant burger',
-        'two human-sized rubber ducks',
-        'a proposal for a new Union Jack',
-        'the recently passed Owning The Internet bill',
-        'the first crewed spaceship to reach Mars',
-        'the largest ever koala recorded in history'
-    ];
-
-    function random(array) {
-        var index = Math.floor(Math.random() * array.length);
-        return array[index];
+    function descriptionPrompt() {
+      return 'Give some context about the image including who, what, where, when and why';
```

Comment:
```suggestion
```

Reply:
will also need to remove the closing `}` manually as that hadn't changed so couldn't 'suggest'